src/Entities/PaymentProcessor.php

```diff
@@ -2,27 +2,22 @@
 
 namespace EscolaLms\Payments\Entities;
 
-use EscolaLms\Payments\Contracts\Billable;
-use EscolaLms\Payments\Dtos\Contracts\PaymentMethodContract;
-use EscolaLms\Payments\Dtos\PaymentDto;
+use EscolaLms\Core\Models\User;
 use EscolaLms\Payments\Enums\Currency;
 use EscolaLms\Payments\Enums\PaymentStatus;
 use EscolaLms\Payments\Events\PaymentCancelled;
 use EscolaLms\Payments\Events\PaymentFailed;
+use EscolaLms\Payments\Events\PaymentRequiresRedirect;
 use EscolaLms\Payments\Events\PaymentSuccess;
-use EscolaLms\Payments\Exceptions\PaymentException;
-use EscolaLms\Payments\Exceptions\RedirectException;
 use EscolaLms\Payments\Facades\PaymentGateway;
 use EscolaLms\Payments\Gateway\Drivers\Contracts\GatewayDriverContract;
 use EscolaLms\Payments\Models\Payment;
-use Illuminate\Database\Eloquent\Model;
+use Illuminate\Http\Request;
 use Omnipay\Common\Message\RedirectResponseInterface;
-use Omnipay\Common\Message\ResponseInterface;
 
 class PaymentProcessor
```

Comment:
Suggest : 
Set the action in the sequence of availability public, protected, private, public static, protected static, private static